### PR TITLE
New article around .NET Core template management and old SDK templates

### LINKED ISSUE #14572: Document how to use .NET Core 3.0 SDK to build an app for the 2.2 runtime
Hi, I don't believe the specified page in its current form answers the following (and IMO pretty basic) question: I am using a .NET Core 3.0  RC1 SDK on my dev machine and am looking to deploy to a Linode node which has the .NET Core 2.2 runtime installed. We know the SDK uses the latest installed SDK version which also (I think?) means the project templates for that SDK version will be used with `dotnet new`. The `mvc` template for example will add NUGet package dependencies to packages targeting .NET Core 3.0 RC1 (because that's my SDK version). They will of course also set the `TargetFramework` to `netcoreapp3.0`. Now I want to build for the .NET 2.2 runtime. I can't just change the TFM to `netcoreapp2.0` because the NuGet packages are still pointing to the 3.0 RC1 versions. Now I could go ahead and change those guys' versions to the .NET Core 2.0 ones, but at this point I'm rewriting so much I'd much rather just use `mvc` template from the .NET 2.2 SDK. It looks to me like I have no such option? Do I really have to reinstall the 3.0 RC1 SDK and install the 2.2 one in order to be able to run `dotnet new mvc` and get the template which has NuGet packages and TFM for the 2.2 runtime?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ab345482-5752-6845-feee-52f0f06c620c
* Version Independent ID: 3613d4c0-be8e-c5a4-42f4-bec78ab99c75
* Content: [Select which .NET Core version to use - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/versions/selection)
* Content Source: [docs/core/versions/selection.md](https://github.com/dotnet/docs/blob/master/docs/core/versions/selection.md)
* Product: **dotnet-core**
* GitHub Login: @Thraka
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by Thraka**
## Summary

- New article on templates via the CLI.
- Described how to install older SDK templates.

Fixes #14572

[Internal review link](https://review.docs.microsoft.com/en-us/dotnet/core/install/templates?branch=pr-en-us-18080)

### CODE REVIEW

File: docs/core/install/templates.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+      Project for some technology (contosoproject) C#
+    Uninstall Command:
+      dotnet new -u /home/username/code/templates
+```
+
+::: zone-end
+
+## Uninstall template
+
+Templates are installed through the [dotnet new](../tools/dotnet-new.md) SDK command with the `-u` parameter. You can either provide the NuGet package identifier of a template, or a folder that contains the template files.
```
Comment (BillWagner) [inline docs/core/install/templates.md:123]: typo?

```suggestion
Templates are uninstalled through the [dotnet new](../tools/dotnet-new.md) SDK command with the `-u` parameter. You can either provide the NuGet package identifier of a template, or a folder that contains the template files.
```

File: docs/core/install/templates.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+| .NET Core 3.0    | [`Microsoft.DotNet.Common.ProjectTemplates.3.0`](https://www.nuget.org/packages/Microsoft.DotNet.Common.ProjectTemplates.3.0) |
+| .NET Core 3.1    | [`Microsoft.DotNet.Common.ProjectTemplates.3.1`](https://www.nuget.org/packages/Microsoft.DotNet.Common.ProjectTemplates.3.1) |
+| ASP.NET Core 2.1 | [`Microsoft.DotNet.Web.ProjectTemplates.2.1`](https://www.nuget.org/packages/Microsoft.DotNet.Web.ProjectTemplates.2.1)       |
+| ASP.NET Core 2.2 | [`Microsoft.DotNet.Web.ProjectTemplates.2.2`](https://www.nuget.org/packages/Microsoft.DotNet.Web.ProjectTemplates.2.2)       |
+| ASP.NET Core 3.0 | [`Microsoft.DotNet.Web.ProjectTemplates.3.0`](https://www.nuget.org/packages/Microsoft.DotNet.Web.ProjectTemplates.3.0)       |
+| ASP.NET Core 3.1 | [`Microsoft.DotNet.Web.ProjectTemplates.3.1`](https://www.nuget.org/packages/Microsoft.DotNet.Web.ProjectTemplates.3.1)       |
+
+For example, the .NET Core SDK includes templates for a console app targeting .NET Core 2.1 and .NET Core 3.1. If you wanted to target .NET Core 3.0, you would need to install the 3.0 templates.
+
+01. Try creating an app that targets .NET Core 3.0.
```
Comment (BillWagner) [inline docs/core/install/templates.md:196]: nit: The leading 0 isn't necessary.

**Review by BillWagner — APPROVED** [review]
This looks great @Thraka 

I think I saw one typo, and I had one other small nit you can decide if you want to address.

Then, :shipit: when ready.

File: docs/core/install/templates.md (hunk 2 above)
Comment (Thraka) [reply docs/core/install/templates.md:196]: I do that so that tabbing is even at 4-spaces in. 😁

**Pull request merged**